internal/bgp/frr/frr.go

```diff
@@ -132,6 +135,19 @@ func (sm *sessionManager) NewSession(l log.Logger, addr string, srcAddr net.IP,
 		advertised:     []*bgp.Advertisement{},
 		sessionManager: sm,
 		bfdProfile:     bfdProfile,
+		multiHop:       multiHop,
+	}
+
+	if s.multiHop == nil {
+		host, _, err := net.SplitHostPort(s.addr)
+		if err != nil {
+			return nil, err
+		}
+
+		s.multiHop, err = ipMultiHopsAway(host)
```

Comment:
I am against changing the default behavior for eBGP the default is single hop away and internally BGP does directly connected check otherwise it won't peer, only when Mutlihop is set bgp will ignore the directly check and I don't think we should ignore it all the time., as I said on slack I think ebgp-multihop is what all other BGP stacks are doing and I think its the right way